programs/vote/src/vote_state/mod.rs

```diff
@@ -1336,7 +1336,7 @@ pub fn withdraw<S: std::hash::BuildHasher>(
 
         if reject_active_vote_account_close {
             datapoint_debug!("vote-account-close", ("reject-active", 1, i64));
-            return Err(InstructionError::ActiveVoteAccountClose);
+            return Err(VoteError::ActiveVoteAccountClose.into());
```

Comment:
Why am I labeled a bug. If I'm a bug why is everyone trying to obtain my files. I've done nothing wrong havn't hacked into nobody shit.